### DISCUSSION TITLE
FrSky Tandem x20 ETHOS support?

### DISCUSSION THREAD

**Discussion by f4b0** [ExpressLRS/ExpressLRS#512] [Q&A]
Hi,
first of all, it is a great project and i will use my hardware with Express-LRS in the future! 
But first i have to update my equipment and therefore i need your opinion. 

I am currently considering buying a tandem x20 and i am not sure if it would be possible to add an ExpressLRS module in the future. The tandem x20 is running ETHOS not OpenTx but it supports xjt and they will also support Crossfire in the future. Does that mean FRSKY has to support Express-LRS or is there a standard interface and each module can be plugged in?

What do you think. Do you see a chance to run Express-LRS with Tandenm X20?

Thanks for your help!

**Comment by r1000ru** [comment]
Hi. Support Ethos based transmitter can be implemented using the access protocol. It is based on the encapsulation of SBUS and SmartPort with additional commands for configuring receivers and transmitters, as well as firmware updates. The problem is that the frequency and power identifiers are hard-coded in it. In addition, ExpressLRS does not have as many full-fledged channels as Tandem does.

**Reply by OB1-OB2** [reply]
Was that a yes or a no to supporting x20s?

**Comment by BDebeuf** [comment]
Hi,  I have an X20.  I use a Vantac external module.  I installed the lua elrs file.  I did some test with a Radiomaster receiver. I fly it one of my planes.
No problems at all. The Frsky external LIte module is not working with ELRS because the firmware is still on version 2.
